src/renderer/router/index.js

```diff
@@ -15,9 +14,8 @@ import Channel from '../views/Channel/Channel.vue'
 import Watch from '../views/Watch/Watch.vue'
 import Hashtag from '../views/Hashtag/Hashtag.vue'
 
-Vue.use(Router)
-
-const router = new Router({
+const router = createRouter({
```

Comment:
According to the docs scrollBehavior changed from using `x` and `y` to using `left` and `top`, so this should probably use those here too: https://router.vuejs.org/guide/migration/#scrollBehavior-changes

sorry, i've looked at that migration page a few too many times 🙈